view/src/components/farms.js

```diff
@@ -358,6 +423,340 @@ class Farms extends Component {
+            label="Farm Tags"
+            placeholder="Type or Select a Farm Tag"
+            InputProps={{
+              ...params.InputProps,
+              startAdornment: (
+                <>
+                  <InputAdornment position="start">
+                    <SearchIcon />
+                  </InputAdornment>
+                  {params.InputProps.startAdornment}
+                </>
+              ),
+            }}
+          />
+        )}
+      />
+    );
+  };
+
+  /** Updates filteredData with a new array; used with filters.js queries */
+  updateCards = (newValues) => {
+    if (newValues === []) {
+      return;
+    }
+    this.setState({
+      filteredData: [...newValues],
+    });
+  };
+
+  /** Reset filteredData to the original page data, upon clicking reset */
+  resetCards = () => {
+    this.setState(
+      {
+        filteredData: [...this.state.data],
+        nameQuery: "",
+        locationQuery: "",
+        tagsQuery: [],
+      },
+      this.populateAllFarmTags
+    );
+  };
+
+  /** Renders filtered produce results into Material-UI cards */
+  handleResultsRender = () => {
+    const { classes } = this.props;
+    const { filteredData } = this.state;
+
+    return (
+      <div>
+        <Grid container spacing={2} alignItem="center">
+          {filteredData.map((farm) => (
+            <Grid item xs={12} key={farm.farmId}>
+              <Card className={classes.root} variant="outlined">
+                <CardContent>
+                  <Typography variant="h5" component="h2">
+                    {farm.farmName}
+                  </Typography>
+                  {farm.farmTags.map((tag) => (
+                    <Chip className={classes.chip} label={tag} size="small" />
+                  ))}
+                  <Box
+                    display="flex"
+                    flexDirection="row"
+                    flexWrap="wrap"
+                    p={0}
+                    m={0}
+                  >
+                    <Box p={3}>
+                      <Typography className={classes.pos} color="textSecondary">
+                        Details:
+                      </Typography>
+                      <Typography variant="body2" component="p">
+                        Location of Farm: {`${farm.location.substring(0, 30)}`}
+                        <br />
+                        {`${farm.location.substring(30, 78)}`}
+                      </Typography>
+                    </Box>
+                    <Box p={3}>
+                      <Typography className={classes.pos} color="textSecondary">
+                        Point of Contact:
+                      </Typography>
+                      <Typography variant="body2" component="p">
+                        Name: {farm.contactName}
+                        <br />
+                        Phone: {farm.contactPhone}
+                        <br />
+                        Email: {farm.contactEmail}
+                      </Typography>
```

Comment:
make sure to merge with updated master which contains the farms.js file that works with a table of contacts (if we merge this it will break because it is now set up for the table)

Reply:
thank you for pointing this out. hmm i'm a bit confused b/c right now git tells me that i am up to date with master when i try to pull, even with some resetting of "origin/master" and attempts of "git fetch". will manually diff this file and master/farms.js and try to fix that way...